src/components/TodoContainer.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import TodosList from "./TodosList";
+import Header from "./Header";
+import InputTodo from "./InputTodo";
+
+class TodoContainer extends React.Component {
+  state = {
+    todos: [
+      {
+        id: 1,
+        title: "Setup development environment",
+        completed: true,
+      },
+      {
+        id: 2,
+        title: "Develop website and add content",
+        completed: false,
+      },
+      {
+        id: 3,
+        title: "Deploy to live server",
+        completed: false,
+      },
+    ],
+  };
+
+  handleChange = (id) => {
+    this.setState((prevState) => ({
+      todos: prevState.todos.map((todo) => {
+        if (todo.id === id) {
+          return {
+            ...todo,
+            completed: !todo.completed,
+          };
+        }
+        return todo;
+      }),
+    }));
+  };
+
+  delTodo = (id) => {
+    this.setState({
+      todos: [
+        ...this.state.todos.filter((todo) => {
+          return todo.id !== id;
+        }),
+      ],
+    });
+  };
+
+  addTodoItem = (title) => {
+    console.log(title);
+  };
+
+  render() {
+    return (
+      <div>
+        <Header />
```

Comment:
Please, make sure to delete the Navbar component file, since you've already decided to use the Header component. Because having empty files will make your project look unnecessarily more crowded.